samples/msal-node-auth-code/index.js

```diff
@@ -2,16 +2,12 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-
-//initialize express
 const express = require("express");
-const app = express();
-const port = 3000;
-app.get('/', (req, res) => redirectToAzureAd(req, res));
-app.get('/redirect', (req, res) =>  acquireToken(req, res));
+const msal = require('@azure/msal-node');
+
+const SERVER_PORT = 3000;
```

Comment:
General best practice is to look for `process.env.PORT`, and fallback to a hard-coded one:

```suggestion
const SERVER_PORT = process.env.PORT || 3000;
```